files/build_templates/sonic_debian_extension.j2

```diff
@@ -246,6 +246,16 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-utilities-data_*.deb || \
 # in bash.bashrc, so we copy a version of the file with it enabled here.
 sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 
+# Remove 'PyGObject' package installed via apt
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove python3-gi
```

Comment:
>remove [](start = 78, length = 6)

replace `remove` with `purge` ?

Reply:
Changed 'remove' to 'purge' for python3-gi